includes/class-give-scripts.php

```diff
@@ -403,6 +403,11 @@ public function admin_localize_scripts() {
 				'processing' => __( 'Processing...', 'give' ),
 				'activating' => __( 'Activating...', 'give' ),
 			),
+			'formTemplate'                      => [
+				'Sequoia' => [
+					'donorChoiceRecurringDonationType' => __( '<strong>Recurring Addon Compatibility Error: </strong>Sequoia form template styling will not look good when you set <code>Recurring donations</code> value to other then <code>Yes - Donor\'s Choice</code>.', 'give' ),
```

Comment:
This isn't a great error message. It should be:

> Error: The form template you have active does not support the Donor's Choice recurring donation method. Please select a different option or change form templates.

Reply:
@DevinWalker we can guide admin which recurring type active form template support. The error message which you suggested does not have information related to that which can cause the donor to update form setting smultiple time.

Reply:
@ravinderk so this message displays when you attempt to activate the form template and you have "admin defined" as a recurring option?

For the error text I have written above I was thinking the Sequoia form template was already active and the admin then configured the recurring option to donor's choice.

In the case of activating the form template and that error handling I would say something like:

> Error: This form template does not support the Recurring Donation choice of Admin Defined. Please update your Recurring Donation option to "Donor's Choice" in order to use this template.

Reply:
fixed: https://github.com/impress-org/givewp/compare/923867334974...99128c5add02